web/themes/custom/hoeringsportal/templates/content/node--citizen-proposal--teaser.html.twig

```diff
@@ -95,4 +95,22 @@
       <span class="activity-teaser-link "><i class="fa-solid fa-arrow-right float-right"></i></span>
     </div>
   </div>
-</a>
+</a> #}
+
+{% extends 'themes/custom/hoeringsportal/templates/components/base-card.html.twig' %}
+
+{% block card_content %}
+  <div class="d-flex my-3 justify-content-between">
+    <div><span>{{ 'End date'|t }}:</span><strong class="ms-1">{{ voteEnd|date('d M. Y') }}</strong></div>
```

Comment:
You should use `format_date('hoeringsportal…` for consistent formatting of dates and times (cf. https://github.com/search?q=repo%3Aitk-dev%2Fhoeringsportal+format_date+path%3A%2F%5Eweb%5C%2Fthemes%5C%2Fcustom%5C%2Fhoeringsportal%5C%2Ftemplates%5C%2F%2F+&type=code).